app/assets/stylesheets/spotlight/_featured_browse_categories_block.scss

```diff
@@ -0,0 +1,69 @@
+.browse-category {
```

Comment:
Good call w/ the variables.  I had was copy-pasting from the example code in the issue and had intended on moving it to variables.  I've added it as a TODO in this PR.

This is using `flexbox` (we're using `.spotlight-flexbox` in the DOM).  I admittedly don't have a full understanding of `flexbox` so we may be doing something redundant or against what `flexbox` gains us here.